tools/generator/src/Logger.swift

```diff
@@ -33,22 +33,24 @@ final class DefaultLogger<E: TextOutputStream, O: TextOutputStream>: Logger {
     }
 
     func logDebug(_ message: @autoclosure () -> String) {
-        // TODO: Colorize
         print("DEBUG: \(message())", to: &self.standardOutput)
     }
 
     func logInfo(_ message: @autoclosure () -> String) {
-        // TODO: Colorize
-        print("INFO: \(message())", to: &self.standardOutput)
+        print("INFO: \(message())".blue, to: &self.standardOutput)
     }
 
     func logWarning(_ message: @autoclosure () -> String) {
-        // TODO: Colorize
-        print("WARNING: \(message())", to: &self.standardError)
+        print("WARNING: \(message())".yellow, to: &self.standardError)
     }
 
     func logError(_ message: @autoclosure () -> String) {
-        // TODO: Colorize
-        print("ERROR: \(message())", to: &self.standardError)
+        print("ERROR: \(message())".red, to: &self.standardError)
```

Comment:
To match Bazel, can we only colorize the prefix? And also, to match Bazel, let's make error bold, info green, debug yellow (as well as warning as it is now).